src/main/java/step2/calculator/domain/OperatorFactory.java

```diff
@@ -0,0 +1,30 @@
+package step2.calculator.domain;
+
+import java.util.Arrays;
+
+public enum OperatorFactory {
+
+	ADDITION(new AdditionOperator()),
+	SUBTRACTION(new SubtractionOperator()),
+	MULTIPLICATION(new MultiplicationOperator()),
+	DIVISION(new DivisionOperator());
+
+	public final static String INPUT_DELIMITER = " ";
+	private final Operator operator;
+
+	OperatorFactory(final Operator operator) {
+		this.operator = operator;
+	}
+	public static Operator findOperator(final String sign) {
+		return Arrays.stream(OperatorFactory.values())
+			.filter(operator -> operator.getOperator().getSign().equals(sign))
```

Comment:
static 메서드에서도 같은 클래스라면 필드 접근이 가능합니다. 그럼 getter를 제거할 수 있을 것 같아요~!

```suggestion
			.filter(operator -> operator.operator.getSign().equals(sign))
```